tools/pm3_amii_bin2eml.pl

```diff
@@ -669,13 +691,15 @@
 __DATA__
 00040402
 01001103
-01000000
-00808000
-92580b4c
-45a9c42f
-a90145ce
-5e5f9c43
-09a43d47
-d232a3d1
-68cbade6
-7f8185c6
+010000
```

Comment:
missing a byte here?

Reply:
It’s intentional; line 670 fills it in with the page count.